src/include/migraphx/op/reduce_sum.hpp

```diff
@@ -13,7 +14,7 @@ namespace op {
 
 struct reduce_sum
 {
-    std::vector<std::size_t> axes;
+    std::vector<int64_t> axes{};
```

Comment:
The type should be `size_t` as the axes cannot be negative.

Reply:
There are negative input in the bert model, so keep it as int64_t

Reply:
Would it be better to reinterpret the negative input when parsing the protobuf? tf reduce_sum also takes in a negative value.

Reply:
I think this is just a design choice.

Reply:
Either way is fine, but we should be consistent across all our operators.

Reply:
We usually use the negative indices in the operators.